### ISSUE TITLE
[BUG]: v1.3.0 throws error with current Google contacts CSV

### ISSUE THREAD

**Issue by nollm** [escalate/google-contacts-birthday-ical-calendar#40]
First of all nice project!
In Germany it is not possible anymore to sync contact birthdays with google calendar because of regulatories.
Your project is a good workaround for that.

Now to the bug that I found.
The release v1.3.0 throws following error with the current Google contacts CSV.
Seems that Google split "Name" to "First Name", "Middle Name" and "Last Name".

```
2024-10-20 11:22:10.740 | INFO     | __main__:cli:81 - CSV file: example.csv
Traceback (most recent call last):
  File "/app/converter.py", line 91, in <module>
    cli()
    ~~~^^
  File "/app/.venv/lib/python3.13/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/app/.venv/lib/python3.13/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/app/.venv/lib/python3.13/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/.venv/lib/python3.13/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/app/converter.py", line 83, in cli
    cal = convert_csv_to_ical(csvfile)
  File "/app/converter.py", line 22, in convert_csv_to_ical
    summary = "{0} hat Geburtstag".format(row["Name"])
                                          ~~~^^^^^^^^
KeyError: 'Name'
```

You corrected that already in the master branch inside `converter.py` because with this it works well.
Maybe you can publish a new release version.

**Comment by escalate** [comment]
Hello @nollm,

Thank you for the tip.
I had forgotten to check in the tests and so there was no release because the pipeline was red.

I will correct this immediately.

Best regards
Felix

**Comment by escalate** [comment]
Fixed with version 1.3.1
